tests/test_bgp_af.rb

```diff
@@ -126,9 +172,13 @@ def test_default_information_originate
     assert(bgp_af.default_information_originate,
            'Error: default-information originate not set')
 
-    pattern = /^ *default-information originate$/
-    af_string = get_bgp_af_cfg(asn, vrf, af)
-
+    if platform == :nexus
+      pattern = /^ *default-information originate$/
+      af_string = get_bgp_af_cfg(asn, vrf, af)
+    elsif platform == :ios_xr
+      pattern = /^ default-information originate$/
+      af_string = @device.cmd("show run router bgp | in default-information")
```

Comment:
Why won't my earlier suggestion of `@device.cmd('show run router bgp #{asn} vrf #{vrf} address-family #{afi} #{safi}')` work for this purpose?